code/modules/uplink/uplink_item_cit.dm

```diff
@@ -7,4 +7,15 @@
 	refundable = TRUE
 	cost = 10
 	surplus = 20 //Let's not have this be too common
+	exclude_modes = list(/datum/game_mode/nuclear)
+
+/datum/uplink_item/stealthy_tools/holoparasite
+	name="Holoparasite Injector"
+	desc="It contains an alien nanoswarm of unknown origin.\
+		  Though capable of near sorcerous feats via use of hardlight holograms and nanomachines.\
+		  It requires an organic host as a home base and source of fuel." //This is the description of the actual injector. Feel free to change this for uplink purposes//
+	item = /obj/item/weapon/storage/box/syndie_kit/holoparasite
+	refundable = TRUE
+	cost = 10 //I'm working off the borer. Price subject to change
```

Comment:
IIRC, the cost was 15 due to the sheer power Guardians have, especially when they can be selected instead of randomized. I suggest keeping it 10 or maybe slightly more because traitor memes, though.